### DISCUSSION TITLE
Without building Docker image, is it still possible to work in the development mode?

### DISCUSSION THREAD

**Discussion by crimson206** [microsoft/autogen#1604] [Q&A]
I tried to build the docker image, but its size was 10gb. All of you experienced the same?
I just wanted to build the image and took it to a cloud service, but because of the size, it seems not possible for me.
If I can't build the dev dockerfile, how can I set up Autogen in development environment?

**Comment by crimson206** [comment]
It is a detouring, but I have an idea. My problem is that, I wanted to run the code in my cloud because of GPU access, however the cloud doesn't allow to use the docker command (pulling is only possible, but not stable.). While I just build the studio Docker in my local pc, I will make my cloud as a free endpoint using temporary web-ui. Therefore, I can set the environment more easily, and I can use the GPU access as well.

**Reply by ekzhu** [reply]
Glad you found a way. Curious what are you building and what's your model setup -- you mentioned GPU access.
